contrib/src/main/java/com/datatorrent/contrib/dimensions/AbstractAppDataDimensionStoreHDHT.java

```diff
@@ -127,7 +127,7 @@
   private Unifier<String> queryResultUnifier;
 
   protected long responseDelayMillis;
-  
+
```

Comment:
@vrozov I'll separate the commit in another PR. Can you point me to some docs that describe how  to do this as malhar jenkins user?